svm/src/account_loader.rs

```diff
@@ -208,30 +208,42 @@ fn load_transaction_accounts<CB: TransactionProcessingCallback>(
         .unique()
         .collect::<Vec<&u8>>();
 
+    let mut fee_payer_account = Some((
+        tx_details.fee_payer_account.data().len(),
+        tx_details.fee_payer_account,
+        tx_details.fee_payer_rent_debit,
+    ));
+
     let mut accounts = account_keys
         .iter()
         .enumerate()
         .map(|(i, key)| {
             let mut account_found = true;
-            #[allow(clippy::collapsible_else_if)]
-            let account = if solana_sdk::sysvar::instructions::check_id(key) {
-                construct_instructions_account(message)
```

Comment:
I guess so, but is there something that prevents the first account from hitting this?

Reply:
Can't sign a transaction as the instruction sysvar and signers are first and there must be at least one signer